lib/widgets.php

```diff
@@ -17,7 +17,12 @@ function gutenberg_is_block_editor() {
 		return false;
 	}
 	$screen = get_current_screen();
-	return ! empty( $screen ) && ( $screen->is_block_editor() || 'gutenberg_page_gutenberg-widgets' === $screen->id );
+	return ! empty( $screen ) &&
+		(
+			$screen->is_block_editor() ||
+			'gutenberg_page_gutenberg-widgets' === $screen->id ||
+			'gutenberg_page_gutenberg-edit-site' === $screen->id
+		);
```

Comment:
Maybe this means this function should move out of lib/widgets.php

Reply:
Not yet, it's still only needed for widget blocks.